src/bootstrap/src/core/build_steps/llvm.rs

```diff
@@ -87,10 +87,14 @@ impl LdFlags {
 ///
 /// This will return the llvm-config if it can get it (but it will not build it
 /// if not).
-pub fn prebuilt_llvm_config(builder: &Builder<'_>, target: TargetSelection) -> LlvmBuildStatus {
-    // If we have llvm submodule initialized already, sync it.
-    builder.update_existing_submodule("src/llvm-project");
```

Comment:
Do we also need to remove this line? Isn't it a problem if we don't resync the submodule before checking for CI download? :thinking: What if `get_closest_merge_commit` works with a non-actual llvm-project checkout?

Reply:
We already sync it here: https://github.com/rust-lang/rust/blob/a3f76a26e045a760bb1163b7eab36872985242d5/src/bootstrap/src/core/config/config.rs#L2740-L2751